src/SimpleCLIUserInterface.cpp

```diff
@@ -0,0 +1,352 @@
+// Copyright (C) Sven Rosvall (sven@rosvall.ie)
+// This file is part of VLCB-Arduino project on https://github.com/SvenRosvall/VLCB-Arduino
+// Licensed under the Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License.
+// The full licence can be found at: http://creativecommons.org/licenses/by-nc-sa/4.0/
+
+#include "SimpleCLIUserInterface.h"
+#include "Controller.h"
+#include <Streaming.h>
+#include <SimpleCLI.h>
+
+extern void printConfig();
+
+
+namespace VLCB
+{
+
+// Create CLI Object
+SimpleCLI simpleCli(CLI_COMMAND_QUEUE_SIZE); // Default of 10 commands
+
+Command cmdHelp;
+Command cmdHelpWithArg;
+Command cmdSerialInterface;
```

Comment:
Can you make these 4 objects members of SimpleCLIUserInterface? 
Avoid global variables.

Reply:
I am not sure what the consequences would be for the way SimpleCLI works  They are in the VLCB namespace. I suggest you adopt the phrase "Avoid global variables where possible" rather than just lecturing me.